app_dart/lib/src/model/proto/internal/scheduler.pbenum.dart

```diff
@@ -1,6 +1,6 @@
 ///
 //  Generated code. Do not modify.
-//  source: lib/src/model/proto/internal/scheduler.proto
+//  source: scheduler.proto
```

Comment:
I'm guessing this was a change to the dart plugin. Tried running the compiler from different working directories, and got the same string.